lib/model.js

```diff
@@ -3143,7 +3143,7 @@ class Model {
       }
       return values;
     }
-    return this.dataValues;
+    return Utils._.clone(this.dataValues);
```

Comment:
@sushantdhiman it's now a different PR, since github doesn't allow to reopen after force push :( 
https://github.com/sequelize/sequelize/pull/8171